### PR TITLE
add page  Login and  Register

### PR DESCRIPTION
**Pull request by mamadykonte**

### CODE REVIEW

File: src/components/Auth/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+  const googleError = () =>
+    alert("Google Sign In was unsuccessful. Try again later");
+  
+   
+    
+
+  return (
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
```
Comment (clelantoson) [inline src/components/Auth/Login.jsx:120]: are you sure you need CssBasline ? its already in App.js

File: src/components/Auth/Register.jsx [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+  },
+  input: {
+    display: "none",
+  },
+  faceImage: {
+    color: theme.palette.primary.light,
+  },
+}));
+
+export default function Register() {
```
Comment (clelantoson) [inline src/components/Auth/Register.jsx:46]:
```suggestion
const Register = () => {
  return ()
};
export default Register;
```
its better to write fonctional component
if you wanted to do the export in one line they explain here its not allowed in ES6 https://stackoverflow.com/questions/41695189/react-how-to-export-a-pure-stateless-component

**Review by clelantoson — CHANGES_REQUESTED** [review]
On the Profile.jsx component I think for now all the social media part is not useful : like , follows, pictures
image

The login register buttons cant appear like this on the profile, and if im not logged I cant see my profile page
image

In the component personal information: on the title, the maj is missing
image

Remember to use these colors in small touches for the design in general https://coolors.co/ffbc42-d7f9ff-3a6ea5-2dd881-426a5a

You can reuse the allergens chips from material UI that you can find on the page /allergens on the component AllergensSelection

Why profile.jsx is not in the same folder ? 
image

For the 404 page I think its useful to write at least "not found" for the user rather than just 404

When i'm signed up my infos are not on my profile
image

You can use this for the personal information, the design doesnt suits to the rest of the app https://material-ui.com/components/text-fields/

The logout button can be displayed in a better way maybe you need to change the css, and not put it on top like this
image

 You can add a margin here under google sign in
image

Normally its "sign in with google"
image

When im logged i shouldnt see the register page
There is a maj on register, we should keep everything in lowcase for the url http://localhost:3000/Register

Avoid to write function Component() its better to write with ES6 syntax : const Component = () => {}

**Review by Dlyf — APPROVED** [review]
Its ok for me. I'm ok with Camille

**Pull request closed without merging**